db/dcrsqlite/apisource.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/decred/dcrdata/rpcutils"
 	"github.com/decred/dcrdata/stakedb"
 	"github.com/decred/dcrdata/txhelpers"
-	humanize "github.com/dustin/go-humanize"
+	"github.com/dustin/go-humanize"
```

Comment:
the package name was repeated, seemingly unnecessarily, because I ran into linters that removed the import claiming it was unused.  One of the reasons I can't stand repos that are not named as the package.